_posts/2017-06-21-pragmatic-architecture-today.md

```diff
@@ -0,0 +1,280 @@
+
+The views are shaped by _perspectives_. 
+These are the cross-cutting concerns that have an impact on the views. 
+Sometimes perspectives are also called quality properties or non-functional requirements:
+
+* Accessibility
+* Availability and Resilience
+* Development resource
+* **Evolution**
+* Internationalisation
+* Location
+* Performance and Scalability
+* Regulation
+* Security 
+* Usability
+
+Again, summaries are available on the website of the book.
+If you want a more in-depth explanation, I really recommend reading the book.
+In today's, agile world, I believe the **Evolution perspective** is a key differentiator in any architectural description.
+Generally, perspectives shape the architecture and deserve the necessary attention.
+
+## Example
+
+<p style="text-align: center;">
+  <img class="image fit" alt="Example" src="/img/prag-arch/f1.jpg">
+</p>
+
+This is the 2017 Mercedes F1 W08 Hybrid. 
+It weights 702kg and has between 750 and 850 horsepower. 
+It is made out of approximately 80 000 different components. 
+The price of the car is an estimated 10 million Eur. 
+That is just for the car, not for the R&amp;D that made the car possible.
+
+Looking back at the viewpoints from above, it is easy to identify how these relate to the construction of the car:
+
+* A Formula One car needs a very specific factory (_Development View_).
+It is not built in the same factory Mercedes builds its trucks.
+* The cars need to be transported all around the world (all the Formula One cars travel over 100 000 miles in the air).
+This can be documented in the _Deployment view_.
+* Maintaining a Formula One car during a race has a huge operational cost and requires a lot of coordination (_Operational View_).
+Just count the number of engineers during a pitstop.
+* ...
+
+In the 2015 and 2016, the predecessors of this car won the Formula One World Championship.
+At the moment of writing, the 2017 car is also leading the championship.
+This pattern is quite common in Formula One.
+The older cars however, are currently up for display in a museum.
+They are rarely used anymore.
+This _throw-away_ approach can also be noticed when comparing to other industries like smartphones or smartwatches.
+A lot of the success of the car, must be its architecture then.
+More specifically, its **ability to change**: to adapt to new rules, competitors and market change.
+If the architecture of a system, has the ability to change, it immediately has a competitive advantage.
+This is especially true in agile projects.
+
+> [Grady Booch](https://nl.wikipedia.org/wiki/Grady_Booch)
+>
+> Architecture represents the significant design decisions that shape a system, where significant is measured by cost of change.
+
+Often it is very difficult to get a system right from the beginning.
+That is why creating a system that has the ability to evolve is important.
```

Comment:
I'd include two commas:
That is why creating a system, that has the ability to evolve, is important.